### ISSUE TITLE
Ability to setup with rnpm

### ISSUE THREAD

**Issue by ahmed1490** [mauron85/react-native-background-geolocation#1]
Amazing work :)
Rnpm will be good dev exp

**Comment by jesperbjerke** [comment]
I used rnpm. Only thing you need to set manually is add the package in MainApplication.java

**Comment by casoetan** [comment]
react-native link works for me on Android.
One thing that breaks is <code>settings.gradle</code>.

Note <code>lib</code> at the end project directory

```java
include ':react-native-mauron85-background-geolocation'
project(':react-native-mauron85-background-geolocation').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-mauron85-background-geolocation/android/lib')
```

**Comment by mauron85** [comment]
@casoetan answer is correct. If you don't like this manual step please "enable subprojects in native Android Components to enable code reuse with cordova":

Link is in the issue:
https://github.com/mauron85/react-native-background-geolocation/issues/126

**Comment by mauron85** [comment]
Implemented in 0.5x branch
